opnode/node/server.go

```diff
@@ -32,23 +34,32 @@ func newRPCServer(ctx context.Context, rpcCfg *RPCConfig, rollupCfg *rollup.Conf
 	// TODO: extend RPC config with options for WS, IPC and HTTP RPC connections
 	endpoint := fmt.Sprintf("%s:%d", rpcCfg.ListenAddr, rpcCfg.ListenPort)
 	r := &rpcServer{
-		endpoint:   endpoint,
-		api:        api,
+		endpoint: endpoint,
+		apis: []rpc.API{{
+			Namespace:     "optimism",
+			Service:       api,
+			Public:        true,
+			Authenticated: false,
+		}},
 		appVersion: appVersion,
 		log:        log,
 	}
 	return r, nil
 }
 
-func (s *rpcServer) Start() error {
-	apis := []rpc.API{{
-		Namespace:     "optimism",
-		Service:       s.api,
-		Public:        true,
+func (s *rpcServer) EnableP2P(backend *p2p.APIBackend) {
+	s.apis = append(s.apis, rpc.API{
+		Namespace:     "",
```

Comment:
I feel like using the `optimism` namespace should be fine here

Reply:
Keeping the p2p server in a "opp2p" namespace for now, since it may not always be active (if p2p is disabled the RPC backend isn't started)